R/calc_DD.R

```diff
@@ -0,0 +1,119 @@
+
+calc_DD_m <- function(tm, k, a, b, t0, beta, c){
+
+  DD_m <- c + beta * tm
+  
+  #when k is missing Tm will always be above the negative temperatures used for k 
+  gt_k <- which(is.na(k) | tm > k)
+  
+  DD_m[gt_k] <- a[gt_k]/(1 + exp(-(tm[gt_k] - t0[gt_k])/b[gt_k]))
+  
+  DD_m
+}
```

Comment:
We can document this function parameter even if it is not exported. Not everyone will know to look in the paper.